vectortile/src/main/scala/geotrellis/vectortile/VectorTile.scala

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2016 Azavea.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package geotrellis.vectortile
+
+import geotrellis.vector._
+import geotrellis.vectortile.protobuf.Value
+
+// --- //
+
+/** A high-level representation of a Vector Tile. At its simplest, a Tile is
+  * just a collection of Layers. We opt to expose each Layer name at the Tile
+  * level, as the keys of a [[Map]]. This way, if the layer names are known by
+  * the user ahead of time, they can search through the Tile quickly.
+  *
+  * Traditionally, VectorTiles are encoded as Protobuf data, which this library
+  * provides a codec for. However, by making this top-level type a trait, we
+  * are able to define alternative backends (GeoJson, for instance).
```

Comment:
Mentioning GeoJson might give users the impression that that is a complete feature